docs/sources/introduction/grafana-enterprise.md

```diff
@@ -21,6 +21,8 @@ To learn more about Grafana Enterprise, refer to [our product page](/enterprise)
 
 Many Grafana Enterprise features are also available in [Grafana Cloud](/docs/grafana-cloud) Free, Pro, and Advanced accounts. For details, refer to [the Grafana Cloud features table](/pricing/#featuresTable).
 
+To Migrate from Grafana Enterprise to Grafana Cloud, Refer to our [Migration guide](/docs/grafana-cloud/account-management/e2c-guide/)
```

Comment:
@ishanjainn , I don't see a migration guide at the location you have specified.

Reply:
https://grafana.com/docs/grafana-cloud/account-management/e2c-guide/
Just got this merged so should be available